app/controllers/idv/in_person/state_id_controller.rb

```diff
@@ -31,7 +31,9 @@ def update
           end
 
           analytics.idv_in_person_proofing_state_id_submitted(
-            **analytics_arguments.merge(**form_result.to_h),
+            **analytics_arguments.
+              merge(**form_result.to_h).
+              merge(form_analytics_properties),
```

Comment:
which form/step is being called here? is it possible to add `birth_year` to the extra analytics properties of that `form_result` instead of having to add this separate `form_analytics_properties`? (the whole purpose behind the `form_result` was to be able to pass per-step information)

Reply:
@zachmargolis Do you mean `form_result` or `form_response`, ie an instance of the `FormResponse` class?  I wanted to clarify because in a later comment you refer to `FormResponse`.

Reply:
The form/step being called is the state_id form submit. I thought about adding the `birth_year` through the `extra_analytics_properties` method, but that would involve either me re-parsing the date (I think this would violate separation of concerns) or adding a new field to the flow_session containing the year. In the `state_id_step` for the flow state machine I added `form_analytics_properties` to the flow session and merged into the extra_analytics_properties when the `extra_analytics_properties` method is called. For the controller step I was trying to be more explicit about what is getting logged which is why I ended up creating the `form_analytics_properties` which should really be the `form_params_analytics_properties` because I am pulling the birth_year from the flow params.

After talking with @eileen-nava, we decided to add the `birth_year` to the `analytics_arguments` method.

Reply:
So after some investigation with @jennyverdeyen (She is working on a similar story) we have decided that `analytics_arguments` won't work because it is used for both the show and update analytics. In the context of show, the `flow_params` method errors out because no params are defined when the show method executes. We didn't feel it was necessary to add extra logic for checking if params are defined or not. So it seemed reasonable to merge the values that need to be logged in the update method where the analytics log is being called.

Tagging: @eileen-nava